README.rst

```diff
@@ -495,6 +499,13 @@ Choose from the list of available fixers:
                         An empty line feed should
                         precede a return statement.
 
+* **scalar_cast** [symfony]
```

Comment:
I would say it's "correct" rather than "short", and the other types are aliases of the correct ones... but... php is so internally inconsistent, i have no idea what they actually intended to be the correct types